change/@fluentui-web-components-057bb4b4-9195-499c-a1c4-4fdb458f1302.json

```
@@ -0,0 +1,7 @@
+{
+  "type": "none",
+  "comment": "",
+  "packageName": "@fluentui/web-components",
+  "email": "joschect@microsoft.com",
+  "dependentChangeType": "patch"
```

```suggestion
  "dependentChangeType": "none"
```
Not sure if this matters as there are no internal dependents on us, but I think none is correct here.